Subject:
Date: Sun Mar 13 23:05:05 CET 2005
From: Vojtech Pavlik <vojtech@suse.cz>
===================================================================
ChangeSet@1.2074, 2005-03-13 23:04:44+01:00, vojtech@suse.cz
  serial: Add support for Broadcom GSM/GPRS/EDGE CardBus modem.
  
  Signed-off-by: Vojtech Pavlik <vojtech@suse.cz>


 8250_pci.c |    9 +++++++++
 1 files changed, 9 insertions(+)


diff -Nru a/drivers/serial/8250_pci.c b/drivers/serial/8250_pci.c
--- a/drivers/serial/8250_pci.c	2005-03-13 23:05:14 +01:00
+++ b/drivers/serial/8250_pci.c	2005-03-13 23:05:14 +01:00
@@ -2229,6 +2229,15 @@
 		pbn_b0_1_115200 },
 
 	/*
+	 * Broadcom CardBus EDGE modem.
+	 */
+
+	{
+		PCI_VENDOR_ID_BROADCOM, 0x4333,
+		PCI_ANY_ID, PCI_ANY_ID,	0, 0,
+		pbn_b0_1_115200 },
+
+	/*
 	 * These entries match devices with class COMMUNICATION_SERIAL,
 	 * COMMUNICATION_MODEM or COMMUNICATION_MULTISERIAL
 	 */
